sdk/tests/CMakeLists.txt

```diff
@@ -1,80 +1,2 @@
-if (APPLE OR WIN32)
-    include(FetchContent)
-    FetchContent_Declare(googletest
-            GIT_REPOSITORY https://github.com/google/googletest.git
-            GIT_TAG release-1.12.1)
-
-    set(INSTALL_GTEST OFF CACHE BOOL "" FORCE)
-    FetchContent_MakeAvailable(googletest)
-    include(googletest)
-else ()
-    find_package(GTest REQUIRED)
-endif ()
```

Comment:
This `FetchContent` could probably just stay in this file and not be repeated in the `unit/` and `integration/` subfolder CMakeLists.txt files

Reply:
Fixed in 517301850c956d3202e3d3446eab1d1a47be2e81.